storage/innobase/log/log0online.cc

```diff
@@ -1710,6 +1714,9 @@ log_online_bitmap_iterator_next(
 		return TRUE;
 	}
 
+	if (i->end_lsn >= i->max_lsn && i->last_page_in_run)
```

Comment:
Not sure - at this point we are past iteration over individual bits stage, we either are finished, either need to read the next page.